turbinia/client.py

```diff
@@ -710,6 +731,92 @@ def format_task_statistics(
     report.append('')
     return '\n'.join(report)
 
+  def format_worker_status(
+      self, instance, project, region, days=0, all_fields=False):
+    """Formats the recent history for Turbinia Workers.
+
+    Args:
+      instance (string): The Turbinia instance name (by default the same as the
+          INSTANCE_ID in the config).
+      project (string): The name of the project.
+      region (string): The name of the zone to execute in.
+      days (int): The number of days we want history for.
+      all_fields (bool): Include historical Task information for the worker.
+    Returns:
+      String of Request status
+    """
+    # Set number of days to retrieve data
+    num_days = 7
+    if days != 0:
+      num_days = days
+    task_results = self.get_task_data(instance, project, region, days=num_days)
+    if not task_results:
+      return ''
+
+    # Sort task_results by last updated timestamp.
+    task_results = sorted(
+        task_results, key=itemgetter('last_update'), reverse=True)
+
+    # Create dictionary of worker_node: {{task_id, task_update,
+    # task_name, task_status}}
+    workers_dict = {}
+    for result in task_results:
+      worker_node = result.get('worker_name')
```

Comment:
I know this is the workers view, but I was wondering if we might want to have an "Unassigned Tasks" or "Scheduled and not assigned" section here (or similar) for Tasks that have been created by the server, but have not been picked up by any worker.  This is most likely to happen when Tasks are already running on all of the workers, so any new Tasks won't get queued on a worker until the existing Tasks are completed.  I think we could probably just set that as the default worker name if there is not a worker name set, and keep everything else the same (except possibly some wording in the report).  LMK what you think.

Reply:
Per our chat, won't implement in this PR.